### DISCUSSION TITLE
Creating new tasks

### DISCUSSION THREAD

**Discussion by vanalljc** [freeplane/freeplane#1769] [Help]
Hi,
I've just started with Freeplane and am looking  to create tasks. I can't find any menu item or support documentation for creating new tasks. Can someone please point me in the right direction?
Thanks,

**Comment by macmarrum** [comment]
Hi @vanalljc 

Welcome to the Freeplane user community!

Freeplane can be used to solve many needs, including task management.

You can have a look at add-ons related to GTD (Getting Things Done):
- https://docs.freeplane.org/getting-started/Add-ons_(install).html

and/or at the following discussion:
- https://github.com/freeplane/freeplane/discussions/478

Please feel free to ask any further questions.

**Reply by vanalljc** [reply]
Hi @macmarrum 
Thanks for the response. I'll definitely check out those resources
Cheers
